UPGRADE_NOTES.md

```diff
@@ -1,3 +1,63 @@
+## Upgrading from 0.26 to 0.27
+
+For a full diff of necessary changes, please see [LuckyDiff](https://luckydiff.com?from=0.26.0&to=0.27.0).
+
+- Upgrade Lucky CLI (homebrew)
+
+```
+brew update
+brew upgrade lucky
+```
+
+- Upgrade Lucky CLI (Linux)
+
+> Remove the existing Lucky binary and follow the Linux
+> instructions in this section
+> https://luckyframework.org/guides/getting-started/installing#on-linux
+
+- Update versions in `shard.yml`
+  - Crystal should be `">= 0.36.1, < 2.0.0"`
+  - Lucky should be `~> 0.27.0`
+  - Authentic should be `~> 0.7.3`
+  - Carbon should be `~> 0.1.4`
+  - Dotenv should be `~> 1.0.0` or replace with [LuckyEnv 0.1.0](https://github.com/luckyframework/lucky_env)
+  - LuckyFlow should be `~> 0.7.3`
+  - JWT (if you use Auth) should be `~> 1.5.1`
+  - LuckyTask needs to be added as a dependency
```

Comment:
This isn't necessary as it will be a transitive dependency from lucky but it doesn't hurt I guess

Reply:
oh right. So maybe this doesn't need to be here? https://github.com/luckyframework/lucky_cli/blob/master/src/generators/web.cr#L166-L168